docs/extending/models.md

```diff
@@ -159,3 +159,20 @@ Dictionary/Map that stores all the additional properties which are not described
 When present, the properties values are store in this backing store instead of using fields for the object. The backing store allows multiple things like dirty tracking of changes, making it possible to get an object from the API, update a property, send that object back with only the changed property and not the full objects. Additionally it will be used for integration with third party data sources.
 
 > **Note:** the backing store is only added if the target language supports it and when the `-b` parameter is passed to the CLI when generating the models.
+
+### Properties and accessors name mangling
+
+To produce a more idiomatic output for specific languages, mangling is applied to the properties names and/or accessors.
+This is the opinionated mangling on each language:
+
+| Language | Property name | Property accessors |
+|---|---|---|
+| CSharp | `PascalCase` | - |
+| Go | - | `PascalCase` |
+| Java | - | `camelCase` |
+| PHP | - | `camelCase` |
```

Comment:
This is at the best of my googling skills, feel free to ask for a different encoding.

Reply:
> Whatever naming convention is used SHOULD be applied consistently within a reasonable scope. That scope may be vendor-level, package-level, class-level, or method-level.

There isn't one way of doing it, it just should be consistent.

https://www.php-fig.org/psr/psr-1/#42-properties

Reply:
Thanks, and PHP is using camelCase https://github.com/microsoft/kiota/blob/2f6bba637cb8cb1b7eae544a2a71bc4b88bd6223/src/Kiota.Builder/Writers/Php/CodePropertyWriter.cs#L13

Reply:
I think we can resolve this one as the encoding in this PR seems to perfectly match the requirement.